src/test-helpers.tsx

```diff
@@ -141,15 +141,98 @@ export const fakeVersion: ExternalVersionWithContent = Object.freeze({
   version: '1.0',
 });
 
+export const fakeExternalDiff = Object.freeze({
```

Comment:
I just moved `fakeExternalDiff` here, so that I can use it in  [`createExternalVersionWithEntries`](https://github.com/mozilla/addons-code-manager/pull/1085/files#diff-e1a04bb40c209dec0edac6868d892215R218).